src/components/Synthetics/ConfirmationBox/SideOrderEntries.tsx

```diff
@@ -101,59 +103,82 @@ function SidecarEntry({
   const removeRowTooltip = useMemo(() => <span>{t`Remove Row`}</span>, []);
 
   return (
-    <div className="SidecarEntry-row" key={entry.id}>
-      <div className={cx("Sidecar-price", { "input-error": priceError })}>
-        <span className="price-symbol">$</span>
+    <div className={cx("flex flex-row gap-10")} key={entry.id}>
+      <div
+        className={cx(
+          "group relative rounded-4 border border-solid border-[#212540] bg-slate-700 pl-5 leading-1 focus-within:border-cold-blue-500 hover:border-cold-blue-700 hover:focus-within:border-cold-blue-500",
```

Comment:
About `[#212540]` is there no similar color in our pallete?

If there is no similar color: is this 212540 color used in multimple places?

is so maybe we could add it to pallete

And BTW you can add newlines in class names to separate them by line